### ISSUE TITLE
Can't restore Sickrage backup

### ISSUE THREAD

**Issue by LucasAdam** [pymedusa/Medusa#3328]
### Before submitting your issue:

Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

**Branch/Commit:**
master/b96b3ee10760ffa03a15a207841a1a2bc65798dd
**OS:**
W10
**What you did:**
Tried to restore sickrage backup & restart
**What happened:**
Medusa asked for restart and never started again
**What you expected:**
All my shows imported
**Logs:**
```
Nov 27 20:46:31 KS-WinZ medusa[2267]: File "/opt/medusa/medusa/__main__.py", line 464, in initialize
Nov 27 20:46:31 KS-WinZ medusa[2267]: Application.restore_cache_folder(app.CACHE_DIR)
Nov 27 20:46:31 KS-WinZ medusa[2267]: File "/opt/medusa/medusa/__main__.py", line 1197, in restore_cache_folder
Nov 27 20:46:31 KS-WinZ medusa[2267]: restore_cache(os.path.join(restore_folder, 'cache'), cache_folder)
Nov 27 20:46:31 KS-WinZ medusa[2267]: File "/opt/medusa/medusa/__main__.py", line 1193, in restore_cache
Nov 27 20:46:31 KS-WinZ medusa[2267]: logger.log(u"Restore: restoring cache successful", logger.INFO)
Nov 27 20:46:31 KS-WinZ medusa[2267]: File "/opt/medusa/medusa/init/logconfig.py", line 34, in __getattr__
Nov 27 20:46:31 KS-WinZ medusa[2267]: return getattr(self.logger, name)
Nov 27 20:46:31 KS-WinZ medusa[2267]: AttributeError: 'Logger' object has no attribute 'INFO'
```

**Comment by adaur** [comment]
https://github.com/pymedusa/Medusa/blob/master/medusa/__main__.py#L1193 seems to be following the old format

**Comment by p0psicles** [comment]
Jip. This one sneaked through, as we're not doing allot of migrations. I'll fix tomorrow.

**Comment by medariox** [comment]
Fixed in develop with: https://github.com/pymedusa/Medusa/pull/3331

**Comment by noodle-** [comment]
@LucasAdam Did you try again after these changes got merged into master?

When I tried the sickrage to medusa migration again, medusa restarted succesfully, the sickrage settings got imported to medusa, but sadly none of the shows did.

Not sure whether it's somehow a problem on my end. Haven't had much time to investigate further by looking at logs etc. I'll create a seperate issue if the migration did work for you.
